app/locales/en.js

```diff
@@ -1059,7 +1059,7 @@ export default {
         },
         furtherHelp: {
           title: '## Further Help',
-          body: 'If you\'d like some further information, check out the [documentation behind building Kitteh Zoo](https://help.zooniverse.org/getting-started/example), that talks you through building this project in the Project Builder.\n\nIf this doesn\'t help, get in contact with the Zooniverse team via the [contact page](/about/contact).',
+          body: 'If you\'d like some further information, check out the [documentation behind building Kitteh Zoo](https://help.zooniverse.org/getting-started/example), that talks you through building this project in the Project Builder.\n\nIf this doesn\'t help, get in contact with the Zooniverse team via the [contact page](https://www.zooniverse.org/about#contact).',
```

Comment:
This is the only translation key that needs editing because the rest that reference a relative path such as `/about/contact` belong to components that will be completely deleted (or at least fully ignored upon launch of the new FEM pages).